frontend/src/scenes/session-recordings/player/inspector/components/ItemEvent.tsx

```diff
@@ -65,6 +66,10 @@ export function ItemEvent({ item }: ItemEventProps): JSX.Element {
             <SummarizeWebVitals properties={item.data.properties} />
         ) : item.data.elements.length ? (
             <AutocapturePreviewImage elements={item.data.elements} />
+        ) : item.data.event === '$ai_generation' ||
+          item.data.event === '$ai_span' ||
+          item.data.event === '$ai_trace' ? (
+            <AIEventSummary event={item.data} />
```

Comment:
ah, i think you need eager loading of the properties for the error flag in here to work...

here we subscribe to web vitals events and preload their event properties https://github.com/PostHog/posthog/blob/3f0512fc70125036eb22bfd64d33585ac53ddedf/frontend/src/scenes/session-recordings/player/sessionRecordingDataLogic.ts#L1165 you can copy that

we delay loading properties in the inspector list until expanded otherwise